homeassistant/components/logentries.py

```diff
@@ -0,0 +1,76 @@
+"""
+Support for sending data to Logentries webhook endpoint.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/logentries/
+"""
+import json
+import logging
+
+import requests
+
+import homeassistant.util as util
+from homeassistant.const import EVENT_STATE_CHANGED
+from homeassistant.helpers import state as state_helper
+from homeassistant.helpers import validate_config
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "logentries"
+DEPENDENCIES = []
+
+DEFAULT_HOST = 'webhook.logentries.com'
+DEFAULT_SSL = False
+
+CONF_TOKEN = 'token'
+CONF_SSL = 'SSL'
+
+
+def setup(hass, config):
+    """Setup the Logentries component."""
+    if not validate_config(config, {DOMAIN: ['token']}, _LOGGER):
+        _LOGGER.error("Logentries token not present")
```

Comment:
`validate_config` is obsolete. Use voluptuous instead and define a `CONFIG_SCHEMA`. I guess that the docs aren't up-to-date.